app/dashboard/views.py

```diff
@@ -518,7 +537,7 @@ def bounty_details(request, ghuser='', ghrepo='', ghissue=0):
                 params['is_legacy'] = bounty.is_legacy  # TODO: Remove this following legacy contract sunset.
                 if profile_id:
                     profile_ids = list(params['interested_profiles'].values_list('profile_id', flat=True))
-                    params['profile_interested'] = request.session.get('profile_id') in profile_ids
+                    params['profile_interested'] = profile_id in profile_ids
```

Comment:
strangely enough it doesnt seem like `profile_interested` is even being used at all in the template

Reply:
Yeah, I noticed that but opted to not yank it out yet.  I can remove it here though.

Reply:
+1